### DISCUSSION TITLE
Flash read error, 1000

### DISCUSSION THREAD

**Discussion by Janco29** [community/community#42659] [Other Feature Feedback, Questions, & Ideas]
Hello. i am getting a flash read error on my TTS-55 laser engraver. Its still brand new and I have no idea o  how to fix it. Could anyone please help??
![Screenshot (93)](https://user-images.githubusercontent.com/121441624/209554956-9342a67f-3340-4403-a136-0d4c385aaa57.png)
I have read up a bit about it, something to do with the firmware on the board.
The name on the board is Makerbase MKS LTS V1.1

**Comment by jfullstackdev** [comment]
I think, if this is correct, they are on GitHub too, so you can ask them there

https://github.com/makerbase-mks

**Comment by Suporte-Speedtech** [comment]
I also need the firmware file for this MKS LTS V1.1 board. 
I can't find it anywhere and the company that made the equipment doesn't distribute the correct firmware
